### DISCUSSION TITLE
32k pre-trained model

### DISCUSSION THREAD

**Discussion by I-i-I-iIi-I-i-I** [SayanoAI/RVC-Studio#13] [General]
Maybe add a pre-trained 32k model for voice training? For example, Mangio and Applio have such a pre-trained model.

**Comment by SayanoAI** [comment]
i'll look into that, thanks. Do you know if the 32k models are objectively better in quality?

**Reply by I-i-I-iIi-I-i-I** [reply]
If I'm not mistaken, if the quality of the dataset in terms of frequency spectrum is less than or equal to 16kHz, a 32k model would be ideal for model training on such a dataset.

**Reply by SayanoAI** [reply]
32k models have been added now
